roles/lobsters/tasks/main.yml

```diff
@@ -76,7 +93,7 @@
 - name: bundle install
   become: true
   become_user: lobsters
-  shell: bin/bundle install --path vendor/bundle
```

Comment:
`command` is used here because the permissions with `become` and `become_user` can get weird according to the docs.